community/dvtm/sources

```diff
@@ -1 +1 @@
-git+https://github.com/martanne/dvtm
+git+https://github.com/martanne/dvtm#7e39b34896f7e1832f0138afc35fcde2377ee198
```

Comment:
Source should be https://github.com/martanne/dvtm/releases/download/v0.15/dvtm-0.15.tar.gz and version should be `0.15 1`